charts/flink-operator/templates/metrics-service.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
     prometheus.io/port: "8443"
     prometheus.io/scheme: https
     prometheus.io/scrape: "true"
+    service.beta.kubernetes.io/aws-load-balancer-additional-resource-tags: kubernetes_namespace={{ .Releases.Namespace }}
```

Comment:
This only makes sense if we actually use a `Service` as a load balancer in our environment.. but we don't for reference. I don't mind adding it, but it's not going to do anything for us.

Reply:
I put in the PR description that we are just spamming the annotation everywhere for now. I can change this to be conditional if we want.